com.unity.render-pipelines.high-definition/Runtime/RenderPipeline/HDRenderPipeline.cs

```diff
@@ -1339,6 +1339,8 @@ void UpdateShaderVariablesGlobalCB(HDCamera hdCamera, CommandBuffer cmd)
                 m_ShaderVariablesGlobalCB._EnableRecursiveRayTracing = 0;
                 m_ShaderVariablesGlobalCB._SpecularOcclusionBlend = 1.0f;
             }
+            
+            m_ShaderVariablesGlobalCB._HybridDeformedVertexStreamIndex = UnityEngine.Time.frameCount & 1;
```

Comment:
I don't think we want to use 
UnityEngine.Time.frameCount here. we have get so many issue with it (https://github.com/Unity-Technologies/Graphics/pull/3173)

We have a FrameCount per Camera. So I guess here we should use camera.GetCameraFrameCount();
cc @adrien-de-tocqueville  / @JulienIgnace-Unity  for confirmation.

Reply:
Indeed, UnityEngine.Time.frameCount is not always incremented in the editor